extensions/amp-bind/amp-bind.md

```diff
@@ -441,34 +441,34 @@ There are several types of runtime errors that may be encountered when working w
 <table>
   <tr>
     <th>Type</th>
-    <th>Example</th>
```

Comment:
It was wrapping poorly on ampproject.org and I think the error message gives enough context.